resources/i18n/en.json

```diff
@@ -207,7 +207,7 @@
         "f.dateIssued.max": "End date",
         "f.subject": "Subject",
         "f.has_content_in_original_bundle": "Has files",
-        "f.entityType": "Entity Type"
+        "f.entityType": "Item Type"
```

Comment:
Note that this `"f.entityType"` key comes from the server, so it can't change until there's a REST PR that fixes it.

Reply:
I created #353 to track those